src/quo/components/profile/expanded_collectible/view.cljs

```diff
@@ -0,0 +1,54 @@
+(ns quo.components.profile.expanded-collectible.view
+  (:require
+    [promesa.core :as p]
+    [quo.components.counter.collectible-counter.view :as collectible-counter]
+    [quo.components.profile.expanded-collectible.style :as style]
+    [quo.foundations.shadows :as shadows]
+    [quo.theme]
+    [react-native.core :as rn]
+    [schema.core :as schema]))
+
+(defn view-internal
+  [{:keys [container-style square? on-press counter image-src]}]
+  (let [[image-size set-image-size] (rn/use-state {})]
+    (rn/use-effect
+     (fn []
+       (p/let [[image-width image-height] (rn/image-get-size image-src)]
+         (set-image-size {:width        image-width
+                          :height       image-height
+                          :aspect-ratio (/ image-width image-height)})))
+     [image-src])
+    ^{:key (str "expanded-collectible-" image-src)}
+    [rn/pressable
+     {:on-press            on-press
+      :accessibility-label :expanded-collectible
+      :style               (merge container-style (style/container square?))}
+     [rn/view {:style {:flex 1}}
+      [rn/image
+       {:style  (merge (shadows/get 2)
+                       {:width         "100%"
```

Comment:
I'm quite confident that to make the image stretch accordingly you need to set width 100% and an aspect ratio so that the height stretches out respective to that.

Reply:
Should we move this to style and maybe add the reasoning for `width "100%"` as a comment there?

Reply:
sorry yeah moved to style folder! 🙏 
I'll add a comment too